src/Dscp/DB/SQLite/Functions.hs

```diff
@@ -0,0 +1,51 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Dscp.DB.SQLite.Functions
+       ( -- * Closing/opening
+         openSQLiteDB
+       , closeSQLiteDB
+       ) where
+
+import Universum
+
+import qualified Database.SQLite.Simple as Lower
+
+import Dscp.DB.SQLite.Class (MonadSQLiteDB (..), MonadSQLiteDBRead (..))
+import Dscp.DB.SQLite.Types (MonadSQLiteCtx, SQLiteDB (..), SQLiteDBLocation (..),
+                             SQLiteParams (..))
+
+-----------------------------------------------------------
+-- Opening/closing
+-----------------------------------------------------------
+
+newtype InvalidSQLitePathException = InvalidSQLitePathException FilePath
```

Comment:
I guess exceptions structure in our sql structure should be more elaborate -- a newtype per exception is a doubtful approach.